source/adapters/level_zero/CMakeLists.txt

```diff
@@ -6,17 +6,17 @@
 
 set(TARGET_NAME ur_adapter_level_zero)
 
-# Copy L0 loader/headers locally to the build to avoid leaking their path.
+# Copy Level Zero loader/headers locally to the build to avoid leaking their path.
 set(LEVEL_ZERO_COPY_DIR ${CMAKE_CURRENT_BINARY_DIR}/level_zero_loader)
-if (DEFINED L0_LIBRARY)
-  get_filename_component(LEVEL_ZERO_LIB_NAME "${L0_LIBRARY}" NAME)
+if (DEFINED EXTERNAL_LEVEL_ZERO_LIBRARY)
```

Comment:
If the intent of `EXTERNAL_LEVEL_ZERO_LIBRARY` is similar to [UR_OPENCL_ICD_LOADER_LIBRARY](https://github.com/oneapi-src/unified-runtime/blob/main/source/adapters/opencl/CMakeLists.txt#L9) to be set at CMake configure time then we should follow the naming convention and also make this a cache variable which a docstring. Maybe something like:

```cmake
set(UR_LEVEL_ZERO_LOADER_LIBRARY "" CACHE FILEPATH "Path of the Level Zero Loader library")
```

And instead of checking if the variable is `DEFINED`, check against `""`:

```cmake
if(NOT UR_LEVEL_ZERO_LOADER_LIBRARY STREQUAL "")
```

Reply:
Yes, these variables have same intent, fixed according to your suggestion, thanks!